app/src/main/java/org/hackillinois/android/view/ProjectAdapter.kt

```diff
@@ -7,7 +7,7 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
```

Comment:
Could you put all three project-related files in the view/ folder into their own subfolder called project or projects, just to match up with the other folders that already exist here.

Reply:
Yep, good call